std_msgs/Bool reset
std_msgs/Int16 drone_id
std_msgs/Header header